source/plugins/ruby/out_mdm.rb

```diff
@@ -102,6 +102,7 @@ def start
           else
             @http_client = Net::HTTP.new(@post_request_uri.host, @post_request_uri.port)
           end
+          @http_client.instance_eval { @ssl_context = OpenSSL::SSL::SSLContext.new(:TLSv1_2) }
```

Comment:
dont we need to build the ruby with openssl to use OpenSSL? the reason I ask is to make sure the changes I am working on work with your updates.

Reply:
ruby is already built with later version of openssl. If mdm is ingestion working in your new bits, then tlsv1_2 is also working. Is that not the case ? Why do we want to build with a different version of openssl ?

Reply:
OK, mdm ingestion works with new bits. IIRC, i remembered you were mentioning about this in scrum so asking whether we would need anything in ruby & openssl combo.